spark-jobs/src/main/resources/reference.conf

```diff
@@ -89,12 +89,20 @@ timezone="UTC"
 #kafka.info.metrics.client.id="controlInfo"
 #kafka.info.metrics.topic.name="control.info"
 
-#kafka.error.client.id="DqErrorSender"
+#kafka.error.client.id="errorInfo"
 #kafka.error.topic.name="dq.errors"
 
 # Optional security settings
 #kafka.security.protocol="SASL_SSL"
 #kafka.sasl.mechanism="GSSAPI"
 
+# SSL configuration specifics
```

Comment:
@dvagapov was thinking to use this when trying to establish a Menas secured Mongo connection. Would it be posible to reuse some of these?

Reply:
I already played with TLS for Kafka. This is a config for Kafka-client.
Mongo client doesn't have that kind of files https://docs.mongodb.com/manual/tutorial/configure-ssl-clients/ only args.